### PR TITLE
fix: creation of Issue Templates for users reference

### PR DESCRIPTION
**Pull request by borntorock**
Fixes https://github.com/jenkins-x/jx-docs/issues/3037

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report
+about: Report a bug encountered with Jenkins X docs
+labels:
+- area/docs
+- kind/bug
```
Comment (borntorock) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: Need to see if these labels for the bug report issue are correct.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```
Comment (borntorock) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Do we still need to have the possibility of creating blank issues for the users in jx-docs repo?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+name: Feature Request
+about: Suggest a/an feature/enhancement to the Jenkins X docs project
+labels:
+- area/docs
+- documentation
```
Comment (borntorock) [inline .github/ISSUE_TEMPLATE/feature_request.md:6]: Are these labels appropriate or should we include `kind/feature` or any other label?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (borntorock) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Apart from Bug Report template and feature request, do we need to also have security policy in this repo as present in jx repo?

**Comment by ankitm123** [comment]
Thinking about it a bit more, since these are docs, I think just a blank template, and a link to discourse (for asking questions - use the yml for it) should be enough imo. wdyt? @borntorock @MarckK

**Comment by MarckK** [comment]
That sounds @ankitm123. 

Linking to discourse for any content issues (which are likely to be most of the docs issues) and issues here for bugs with the docs site or suggested improvements to its structure, etc.

**Comment by borntorock** [comment]
I have created these templates with reference to the Kubernetes website / docs repo. See [here](https://github.com/kubernetes/website/issues/new/choose). 

A blank template would be similar to what we have currrently. IMHO, I think a template should be available for the users (for reference) while opening up a bug for docs site. Though following the template strictly is not a compulsion for the users.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (ankitm123) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Only blank is what we should have imo. Anything more sounds excessive to me honestly.

Comment (borntorock) [reply .github/ISSUE_TEMPLATE/config.yml:1]: @MarckK What do you think? Let me know, so as to finalize this PR.

**Comment by ankitm123** [comment]
If you see some of the issues in the kubernetes website, I really dont see the value in those templates, as people just ignore them :smile: The most recent ones are the one to do with translation, which we are not mature or big enough to implement imo. 

However, I am ok with making it exactly like the kubernetes doc repo issue section. May be we can see what the user does, and if ppl follow it, stick to it, if not, then remove it. wdyt?

Also, you need to change the commit message and PR title format.

**Comment by borntorock** [comment]
@ankitm123 Sounds good. But  I don't think people will actually and always follow the template. But this is just a medium to gather as much information as we can regarding the bug or feature request they are making. 

On the other hand, I understand your concern and since these templates doesn't enforce the users to follow the template strictly, they can definitely overwrite and proceed with the creation of blank issue.

**Comment by borntorock** [comment]
@ankitm123 Could you Please provide appropriate example for this `Also, you need to change the commit message and PR title format.` I'm not sure why the semantic checks are failing. I tried changing the commit message as well as PR title still some checks are not successfull. Thanks!

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (ankitm123) [reply .github/ISSUE_TEMPLATE/config.yml:1]: We should have blank issues.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (ankitm123) [reply .github/ISSUE_TEMPLATE/feature_request.md:6]: `area/docs` and `documentation` labels in a docs repo dont add much information.
I prefer `kind/feature` (or `kind/enhancement`) and `kind/bug`.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ankitm123) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: Remove the area/docs, not required imo. Every issue opened here would be area/docs.

**Comment by ankitm123** [comment]
@borntorock I am fine with the extra feature and bug templates. Left a few comments for you to address.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (borntorock) [reply .github/ISSUE_TEMPLATE/feature_request.md:6]: you mean `kind/bug` for both bug report and feature request?

**Comment by borntorock** [comment]
just addressed your comments and changed accordingly. Let me know, if this looks good to  be merged now.

**Pull request merged**